AllReadyApp/Web-App/AllReady/Features/Notifications/VolunteerAssignedToItineraryHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AllReady.Models;
+using MediatR;
+using Microsoft.Data.Entity;
+
+namespace AllReady.Features.Notifications
+{
+    public class VolunteerAssignedToItineraryHandler : IAsyncNotificationHandler<VolunteerAssignedToItinerary>
+    {
+        private readonly AllReadyContext _context;
+        private readonly IMediator _mediator;
+
+        public VolunteerAssignedToItineraryHandler(AllReadyContext context, IMediator mediator)
+        {
+            _context = context;
+            _mediator = mediator;
+        }
+
+        public async Task Handle(VolunteerAssignedToItinerary notification)
+        {
+            var taskSignup = await _context.TaskSignups.SingleAsync(ts => ts.Id == notification.TaskSignupId).ConfigureAwait(false);
```

Comment:
Probably should get the User include added as I'm fairly sure we'll need that. I'll have to try and work out why on my run through I'm not seeing all of the data.